### DISCUSSION TITLE
image_shape parameter

### DISCUSSION THREAD

**Discussion by ciberola** [PaddlePaddle/PaddleOCR#7820] [Q&A]
Hi Paddle Enthusiasts! 
I would like to aks about parameter image_shape. 
By default in yml file it is set as:
  "image_shape: [3, 48, 320]", 
but in documentation about training (https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.6/doc/doc_en/recognition_en.md) during inference tests we see command: 
python3 tools/infer/predict_rec.py --image_dir="./doc/imgs_words_en/word_336.png" --rec_model_dir="./your inference model" --rec_image_shape="3, 32, 100" --rec_char_dict_path="your text dict path"
Can someone explain me in detail, what is this parameter used for?

**Comment by andyjiang1116** [comment]
it depends on which model you used, for PP-OCRv3 rec model, the input image shape is 3,48,320, for CRNN model, the input shape is 3,32,100. you can set this param to fit your model, usually it's same with your training config.

**Comment by sheltonsuen** [comment]
I'm also wondering what is this parameter for, and I can see if the rec_image_shape not consitent with your traning config, the predict result of the modle sometime will miss some words
